fabric-convention-tags-v2/src/generated/resources/assets/fabric-convention-tags-v2-datagen/lang/en_us.json

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
Add this to crowdin.yml, and make the translated files go into /src/main/resources and not /generated/resources.

Reply:
Is there a way to make datagen output to the src resource folder instead of generated? I'm not good with datagen so wasn't sure if that was even an option

Reply:
No, I would keep the en_us file in generated, but all of the translated files can go into the main sourceset. When built into a jar they will be next to each other.

Reply:
Ah ok. So in context of this PR, I would leave the datagen side alone. Just add the entry to crowdin.yml that points to `/src/main/resources` and that's it for this part

Reply:
Yes 👍

Reply:
I think I got the crowdin entry right now. Lmk if it is incorrect

Reply:
I have just realised that the location of this file is wrong, as well, its currently:

`fabric-convention-tags-v2/src/generated/resources/assets/fabric-convention-tags-v2-datagen/lang/en_us.json`

when it should be:

`fabric-convention-tags-v2/src/generated/resources/assets/fabric-convention-tags-v2/lang/en_us.json`


This can be fixed by overriding `getEffectiveModId()` in the datagen entrypoint.

Reply:
This still needs fixing.

Reply:
Fixed in [a38fbab](https://github.com/FabricMC/fabric/pull/3716/commits/a38fbab45b56e126ceb22e3c5802b422a44087f5) now. Should be ok.
![image](https://github.com/FabricMC/fabric/assets/40846040/c1aa4fdc-e611-4304-8d5b-01d6c4b7b206)
![image](https://github.com/FabricMC/fabric/assets/40846040/2194957a-5d90-4d2c-ac83-c067df43e162)